### PR TITLE
Add info on has_many association extra option

### PR DESCRIPTION
**Pull request by binarygit**
Docs for https://github.com/avo-hq/avo/pull/3048

output:
image

### CODE REVIEW

**Review by adrianthedev — CHANGES_REQUESTED** [review]
This needs renaming too.

Let's show the full field declaration and shoa a screenshot with the result.

**Comment by binarygit** [comment]
Done @adrianthedev

**Review by adrianthedev — CHANGES_REQUESTED** [review]
Thanks for the rename.

Let's also show the full field declaration and show a screenshot with the result.

The user will get much more context of of that.

File: docs/3.0/associations/has_many.md [hunk 1]
```diff
@@ -43,6 +43,21 @@ field :members,
   as: :has_many,
   through: :memberships
 ```
+<Option name="`attach_fields`">
+
```
Comment (Paul-Bob) [inline docs/3.0/associations/has_many.md:47]:
```suggestion
<VersionReq version="3.11" />
```

**Comment by binarygit** [comment]
@Paul-Bob I updated the gif and also the PR description :pray:

**Review by Paul-Bob — APPROVED** [review]

**Pull request merged**